lib/libgtkwave/src/gw-fst-loader.c

```diff
@@ -753,15 +753,15 @@ static GwDumpFile *gw_fst_loader_load(GwLoader *loader, const char *fname, GErro
     self->subvar_jrb = make_jrb(); /* only used for attributes such as generated in VHDL, etc. */
     self->synclock_jrb = make_jrb(); /* only used for synthetic clocks */
 
-    uint numfacs = fstReaderGetVarCount(self->fst_reader);
+    uint64_t numfacs = fstReaderGetVarCount(self->fst_reader);
```

Comment:
GTKWave and libgtkwave use the integer types that are provided by [GLib](https://docs.gtk.org/glib/types.html) instead of the `stdint.h` ones:
```suggestion
    guint64 numfacs = fstReaderGetVarCount(self->fst_reader);
```
This doesn't apply to the other libraries, like `libfst` which don't depend on GLib/GTK.